core/templates/pages/admin-page/navbar/admin-navbar.component.html

```diff
@@ -4,7 +4,6 @@
     background-color: #00376d;
     color: #fff;
   }
-
```

Comment:
I don't think this has been answered.

Reply:
Greetings! 
@Lawful2002 the reason I removed that was due to lint checks , As you know that lint checks would fail on space or trailing spaces and also last time a made mistake and you guided me regarding the lexicographical ordered letters to , due to lint checks I have removed because trailing space resulted in lint check failure